### PR TITLE
Automate collection of ONT QC metrics

### PR DESCRIPTION
**Pull request by kedhammar**
For a new sequencing run, the PromethION script will now go through the instrument logs to find all previous pore counts recorded for the running flowcell, either from flow cell QC or sequencing MUX scans. This information will be dumped in the sequencing run directory (same as the run_path.txt).

TACA will then parse this information and append it to the CouchDB entry.

:)

### CODE REVIEW

**Comment by kedhammar** [comment]
TODO 

- update the PromethION CRON job to 
`13 * * * * python3 ~/TACA-master/taca/nanopore/promethion_transfer.py --source /data/data-offload --dest /srv/ngi_data/sequencing/promethion --archive /data/nosync --minknow_logs /var/log/minknow/`

-  update AM1620

File: taca/nanopore/promethion_transfer.py [hunk 1]
```diff
@@ -109,13 +113,152 @@ def archive_finished_run(run_dir, archive_dir):
     else:
         print("Some data is still left in {}. Keeping it.".format(top_dir))  # Might be another run for the same project
 
+
+def parse_position_logs(minknow_log_dir: str) -> list:
+    """Look through all flow cell position logs and boil down into a strucutred list of dicts
```
Comment (aanil) [inline taca/nanopore/promethion_transfer.py:118]:
```suggestion
    """Look through all flow cell position logs and boil down into a structured list of dicts
```

File: taca/nanopore/promethion_transfer.py [hunk 2]
```diff
@@ -109,13 +113,152 @@ def archive_finished_run(run_dir, archive_dir):
...
+            type = "qc"
+        elif "INFO: mux_scan_result (user_messages)" in entry["category"]:
+            type = "mux"
+        else:
+            type = "other"
+
+        if type in ["qc", "mux"]:
+
+            new_entry = {
+                "flow_cell_id": entry["body"]["flow_cell_id"],
```
Comment (aanil) [inline taca/nanopore/promethion_transfer.py:187]: Will the entries always have a body if they are qc or mux? There was a `if "body" not in entry:` in `parse_position_logs`

**Review by aanil — APPROVED** [review]
👍 Looks good! Just some minor comments.

File: taca/nanopore/promethion_transfer.py (hunk 2 above)
Comment (kedhammar) [reply taca/nanopore/promethion_transfer.py:187]: Yes, some logs are one-liners and do not have a body but QC and MUX always seem to.

**Comment by kedhammar** [comment]
Waiting on IT to remount Prom before deploying

**Pull request merged**